conf/pipeline/eager/maestro.config

```diff
@@ -0,0 +1,107 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running nf-core eager on whole genome data or mitogenomes 
+ * -------------------------------------------------
+ * nextflow run nf-core/eager -profile maestro,<qos>,maestro,<genome> (where <qos> is long or normal and <genome> is nuclear or mitocondrial)
+ */
+
+params {
+
+   config_profile_name = 'nf-core/eager nuclear/mitocondrial - human profiles'
+
+   config_profile_description = "Simple profiles for human nuclear or mito genomes"
+
+   email = 'pluisi@pasteur.fr'
```

Comment:
This might be a bit dangerous, it will mean ANYONE using this profile at Pasteur will have their results file sent to your email ;)